pkg/controller/azvolumeattachment.go

```diff
@@ -147,64 +149,102 @@ func (r *reconcileAzVolumeAttachment) GetReplicaCount(ctx context.Context, under
 }
 
 // ManageAttachmentsForVolume will be runing on a separate channel
-func (r *reconcileAzVolumeAttachment) SyncAll(ctx context.Context) error {
+func (r *reconcileAzVolumeAttachment) SyncAll(ctx context.Context) (bool, error) {
```

Comment:
Non blocking, but we should probably do a scrub of function names and make internal only functions package private (i.e. start with a lower-case letter).